### ISSUE TITLE
[SUGGESTION] AI denoise/upscaling

### ISSUE THREAD

**Issue by forart** [Intel-Media-SDK/MediaSDK#2566]
It would be extremely useful take advantage of Intel's CPUs powerfulness to "clean up" and upscale old/small images and videos for contemporary usage.

Here are some - maybe - interesting open source resources:

**DENOISE**
- [OpenImageDenoise](https://github.com/OpenImageDenoise/oidn), of course...
- [BasicSR](https://github.com/xinntao/BasicSR): an open source image and video restoration toolbox based on PyTorch, such as super-resolution, denoise, deblurring, JPEG artifacts removal, etc.
- [Kai Zhang's repos](https://github.com/cszn): Image Restoration; Inverse Problems
- [MIRNet](https://github.com/tuvovan/MIRNet---Keras): Keras Implementation of MIRNet - SoTA in Image Denoising, Super Resolution and Image Enhancement
- [Deep Plug-and-Play Image Restoration](https://github.com/JiaxiongQ/Denoise): Deep Plug-and-Play Image Restoration (PyTorch)
- [FastDVDnet](https://github.com/m-tassano/fastdvdnet): A state-of-the-art, simple and fast network for Deep Video Denoising which uses no motion compensation.
- [FastTVD](https://github.com/Michael-Shamouilian/FastTVD): Fast Total Variation Denoising and Speckle Denoising Utilizing GPUs and Multi-Core CPUs
- [CondatTV](https://github.com/MarieRoald/condat_tv): Cython wrapper for the efficient TV denoising algorithm by [Laurent Condat](https://lcondat.github.io/).
- [Implementation of Stochastic Image Denoising](https://github.com/pornosaur/stochastic-image-denoising)
- [Baseline Image Denoiser](https://github.com/Sohambasu07/Baseline_Image_Denoiser): Baseline Autoencoder based Denoiser with symmetric skip connections
- [image-denoise](https://github.com/matlabf/image-denoise): to remove noise from a noisy image, so as to restore the true image.
- [Deep Residual Autoencoder for Real Image Denoising](https://github.com/yilmazdoga/Deep_Residual_Autoencoder_for_Real_Image_Denoising)
- [Matlab-Image-Denoise](https://github.com/AomanHao/Matlab-Image-Denoise)
- [Awesome-Denoise](https://github.com/oneTaken/Awesome-Denoise): All latest image/burst/video Denoising papers with code & citation published in top conference and journal.

**UPSCALING**
- [Video2X](https://video2x.org/): Machine learning video/GIF/image upscaling
- [Waifu2x-Extension-GUI](https://github.com/AaronFeng753/Waifu2x-Extension-GUI): Image, GIF and Video enlarger/upscaler(super-resolution) achieved with Waifu2x, SRMD, RealSR, Anime4K and ACNet.
- [Video Super Resolution](https://github.com/LoSealL/VideoSuperResolution): A collection of state-of-the-art video or single-image super-resolution architectures, reimplemented in tensorflow.
- (https://github.com/Alexkral/AviSynthAiUpscale): An AviSynth+ implementation of some Super-Resolution Convolutional Neural Networks.
- [Fast-SRGAN](https://github.com/HasnainRaz/Fast-SRGAN): enable real time super resolution for upsampling low resolution videos.
- [Anime4K](https://github.com/bloc97/Anime4K): a set of open-source, high-quality real-time anime upscaling/denoising algorithms that can be implemented in any programming language.
- [Zooming-Slow-Mo](https://github.com/Mukosame/Zooming-Slow-Mo-CVPR-2020): Fast and Accurate One-Stage Space-Time Video Super-Resolution
- [Video-Super-Resolution](https://github.com/flyywh/Video-Super-Resolution) list: collection of papers

Hope that inspires !
